CHANGELOG.md

```diff
@@ -10,17 +10,25 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Added
 
-- Support queue 1.2.0 (#177)
+### Changed
+
+### Fixed
+
+## [1.9.0] - 2022-10-05
+
+The release adds support for the latest version of the
```

Comment:
Do we write short summary messages in CHANGELOG now?

Reply:
Before the pull request we have such messages for 1.5.0 and 1.6.0. Summary messages were missed for 1.7.0 and 1.8.0.

We can unify it here.